src/card/lib/card-utils.js

```diff
@@ -304,6 +304,18 @@ export function filterExtraFields(extraData : Object) : ExtraFields | Object {
     }, {});
 }
 
+
+export function parsedCardType(potentialCardTypes: Array<object>) : Array<object> {
+    const parsedCardTypeInfo = [];
+    potentialCardTypes.forEach((card) => {
+        const { type, niceType, code } = card;
+        parsedCardTypeInfo.push( { type, niceType, code })
+    })
+
+    return parsedCardTypeInfo;
```

Comment:
not urgent, but you could re-write this as
```suggestion
    return potentialCardTypes.map(({ type, niceType, code }) => ({
      type, niceType, code
    }));
```